src/utils/GitHubUtils/GitHubUtils.js

```diff
@@ -61,56 +61,58 @@ export const setEmptyCalendarValues = calendar => {
   return copiedCalendar;
 };
 
-export const mergeCalendarsContributions = (
-  actualCalendar,
-  gitHubUserJsonCalendar,
-  timeframe
-) => {
+export const sanitize = calendar =>
+  calendar.children[0].children
+    .filter(item => {
+      const isDay =
+        item.attributes.class !== "wday" && item.attributes.class !== "month";
+
+      return isDay;
+    })
+    .map(week => week.children.map(day => day));
+
+export const aggregateCalendars = (actualCalendar, userCalendar) => {
   const copiedActualCalendar = javaScriptUtils.deepCopyObject(actualCalendar);
 
-  gitHubUserJsonCalendar.children[0].children.forEach(
-    (weeklyData, weekIndex) => {
-      weeklyData.children
-        .filter(dailyData =>
-          calendarUtils.filterContributionDays(dailyData, timeframe)
-        )
-        .forEach((dailyData, dayIndex) => {
-          const actualCalendarDailyData = calendarUtils.getCalendarDataByIndexes(
-            copiedActualCalendar,
-            weekIndex,
-            dayIndex
-          );
-          const totalDailyContributions =
-            Number(actualCalendarDailyData.attributes["data-count"]) +
-            Number(dailyData.attributes["data-count"]);
-
-          copiedActualCalendar.children[0].children[weekIndex].children[
-            dayIndex
-          ].attributes = {
-            ...actualCalendarDailyData.attributes,
-            "data-count": String(totalDailyContributions),
-            fill: calendarUtils.getFillColor(totalDailyContributions)
-          };
-        });
-    }
+  userCalendar.forEach((week, weekIndex) =>
+    week.forEach((day, dayIndex) => {
+      if (day) {
+        const actualCalendarDailyData = calendarUtils.getCalendarDataByIndexes(
+          copiedActualCalendar,
+          weekIndex,
+          dayIndex
+        );
+
+        const totalDailyContributions =
+          +actualCalendarDailyData.attributes["data-count"] +
```

Comment:
The explicit cast to Number was easier to read imo.

Reply:
https://github.com/c-hive/team-contribution-calendar/pull/70/commits/dcd7948a18ff70fef7737a8f88f2fac8fe5feb47